ssawg_trending_scraper.py

```diff
@@ -166,276 +179,329 @@ def get_url(self):
 
             if page_request.status_code == 200:
                 # use astropy Table
-                table_page = (Table.read(page_request.text, format='ascii.html',
-                                         htmldict={'table_id': 2}))
+                table_page = Table.read(
+                    page_request.text, format="ascii.html", htmldict={"table_id": 2}
+                )
                 # pull quarterly start and stop dates from page
-                start_time = DateTime(table_page['TSTART'][0])
-                stop_time = DateTime(table_page['TSTOP'][0])
+                start_time = DateTime(table_page["TSTART"][0])
+                stop_time = DateTime(table_page["TSTOP"][0])
                 # define halfway through the quarter
                 halfway = start_time.secs + ((stop_time.secs - start_time.secs) / 2)
                 # is now > 50% through quarter?
                 if DateTime().secs > halfway:
-                    return f'{URL_ASPECT}/{self.page}/{year}/Q{quarter}/', ''
+                    return f"{URL_ASPECT}/{self.page}/{year}/Q{quarter}/", ""
                 # if not 50% through and it's the first quarter of the year
                 elif quarter == 1:
                     # switch to fourth quarter of previous year
-                    return (f'{URL_ASPECT}/{self.page}/{year-1}/Q4/',
-                            f'{URL_ASPECT}/{self.page}/{year}/Q{quarter}/')
+                    return (
+                        f"{URL_ASPECT}/{self.page}/{year-1}/Q4/",
+                        f"{URL_ASPECT}/{self.page}/{year}/Q{quarter}/",
+                    )
                 else:
                     # try previous quarter
-                    return (f'{URL_ASPECT}/{self.page}/{year}/Q{quarter-1}/',
-                            f'{URL_ASPECT}/{self.page}/{year}/Q{quarter}/')
+                    return (
+                        f"{URL_ASPECT}/{self.page}/{year}/Q{quarter-1}/",
+                        f"{URL_ASPECT}/{self.page}/{year}/Q{quarter}/",
+                    )
             else:
                 continue
 
-        raise RuntimeError(f'failed to find URL for {self.page}')
+        raise RuntimeError(f"failed to find URL for {self.page}")
+
+
+class AcqStatReportsPage(ReportsPage):
+    page = "acq_stat_reports"
+
+    def get_html_chunks(self):
+        html_chunks = [
+            self.headers2[0],
+            self.url_html,
+            self.headers3[0],
+            self.tables[1],
+            self.current_quarter_acq_images["id_acq_stars.png"],
```

Comment:
It looks like this id_acq_stars.png is ending up on the output page without the full URL, so is just showing up as a broken image.  I think this is not a regression, but could have a fix in this PR anyway.

Reply:
Fixed.